### ISSUE TITLE
Discussion threads for digital surface twins

### ISSUE THREAD

**Issue by tdbjacobs** [ContactEngineering/topobank#744]
Dear all, 

Not sure if this is a good idea, or even possible, but it might be useful if we could have on-going discussion attached to a surface, including the ability to @mention people. 

For example, Luke just shared his surface and asked for review from Antoine and me; all the better if he could have requested that in the web app. 

But maybe this is too complicated. 

Take care,

Tevis

**Comment by pastewka** [comment]
Yes, but that would be hard to integrate into contact.engineering

**Comment by pastewka** [comment]
Not sure if this even needs to be a proper chat, a discussion board (like this issue tracker here) would suffice.

**Comment by pastewka** [comment]
We may want to checkout django-comments: https://github.com/django/django-contrib-comments
